### PR TITLE
raidboss: Basic FRU P1 triggers

### PR DESCRIPTION
**Pull request by JLGarber**
This is partially based on an experimental set of triggers @valarnin tossed in the ACT Discord server yesterday or the day before. I wasn't able to get the Blasting Zone safe area triggers working, but apart from that and the tank buster trigger, everything in here is stuff I was able to test tonight and confirm as working. The timing for delays and durations is all stuff that fits relatively closely to match mechanics. (Castbars in phase 1 are a little longer than we're used to.)

### CODE REVIEW

**Comment by valarnin** [comment]
> Blasting Zone safe area

Did you mean the Utopian Sky safe spots? That set of triggers should have been pretty simple/not prone to failure, and I didn't have any issues with my WIP stuff during in-game testing tonight.

Clones spawn at the start of the fight, then during Utopian Sky they get their position set, and then get an ActorControlExtra line to play the animation for arm lifting.

Here's an example of the lines for one actor:

```
03|2024-11-27T23:19:14.3620000-05:00|400289E8|Fatebreaker's Image|
26|2024-11-27T23:20:34.8500000-05:00|655|Concealed|9999.00|E0000000||400289E8|Fatebreaker's Image|01|188300|
271|2024-11-27T23:20:34.8500000-05:00|400289E8|-1.5709|00|00|120.0000|100.0000|0.0000|
273|2024-11-27T23:20:36.9860000-05:00|400289E8|003F|4|0|0|0|
```

There's enough delay between the 271 and 273 lines that there shouldn't ever be a case of out-of-order lines, I think...

**Comment by JLGarber** [comment]
I'm not sure. I saw exactly one successful activation on those triggers yesterday while running, and it was north/south safe. Unfortunately I wasn't in a position to stop and debug things since we were chain pulling, so I didn't get to see whether there were any errors in the ACT console. I'll give it another attempt tonight and try to do some scrollback during our break maybe.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts [hunk 1]
```diff
@@ -1,15 +1,203 @@
...
+      durationSeconds: 8,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Line Cleave -> Knockback',
+          de: 'Linien AoE -> Rückstoß',
+          fr: 'AoE en ligne -> Poussée',
+          ja: '直線範囲 -> ノックバック',
+          cn: '直线 -> 击退',
+          ko: '직선 장판 -> 넉백',
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts:96]:
```suggestion
          en: 'Line Cleave => Knockback',
          de: 'Linien AoE => Rückstoß',
          fr: 'AoE en ligne => Poussée',
          ja: '直線範囲 => ノックバック',
          cn: '直线 => 击退',
          ko: '직선 장판 => 넉백',
```
We should keep these consistent.

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts [hunk 2]
```diff
@@ -1,15 +1,203 @@
...
+      durationSeconds: 8,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Line Cleave -> Out',
+          de: 'Linien AoE -> Raus',
+          fr: 'AoE en ligne -> Extérieur',
+          ja: '直線範囲 -> 離れる',
+          cn: '直线 -> 去外侧',
+          ko: '직선 장판 -> 바깥으로',
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts:113]:
```suggestion
          en: 'Line Cleave => Out',
          de: 'Linien AoE => Raus',
          fr: 'AoE en ligne => Extérieur',
          ja: '直線範囲 => 離れる',
          cn: '直线 => 去外侧',
          ko: '직선 장판 => 바깥으로',
```

File: ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts [hunk 3]
```diff
@@ -1,15 +1,203 @@
...
+      type: 'StartsUsing',
+      netRegex: { id: '9CD7', source: 'Fatebreaker\'s Image', capture: false },
+      durationSeconds: 10,
+      infoText: (_data, _matches, output) => output.fireSafe!(),
+      outputStrings: {
+        fireSafe: {
+          en: 'Fire Safe',
+        },
+      },
+    },
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts:140]:
```suggestion
    },
    {
      id: 'FRU P1 Burnished Glory',
      type: 'StartsUsing',
      netRegex: { id: '9CEA', source: 'Fatebreaker', capture: false },
      response: Responses.bleedAoe(),
    },
```

**Comment by xiashtra** [comment]
> I'm not sure. I saw exactly one successful activation on those triggers yesterday while running, and it was north/south safe. Unfortunately I wasn't in a position to stop and debug things since we were chain pulling, so I didn't get to see whether there were any errors in the ACT console. I'll give it another attempt tonight and try to do some scrollback during our break maybe.

For what it's worth, when I tested @valarnin's initial triggers in-instance they all worked without issue.

**Comment by JLGarber** [comment]
Quickly pushing during break with issues addressed. I'm not sure what I did wrong when I originally tested the Utopian Sky triggers, but re-adding them worked this time.

**Review by xiashtra — APPROVED** [review]
LGTM.  We might want to add some comments separating the different phases so people can work in different parts of the same file without causing a lot of merge conflicts.

**Pull request merged**